apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -546,6 +553,8 @@ const EventTypePage = (props: EventTypeSetupProps & { allActiveWorkflows?: Workf
   const handleSubmit = async (values: FormValues) => {
     const { children } = values;
     const dirtyValues = getDirtyFields(values);
+    console.log("HI-------");
+    console.log(dirtyValues);
```

Comment:
remove this

Reply:
Ohooo.. my bad.. Super embarrassing.. 🫣😅